CRUDRecipeEF.BL.DL/Services/RecipeService.cs

```diff
@@ -7,18 +7,23 @@
 using CRUDRecipeEF.BL.DL.DTOs;
 using CRUDRecipeEF.BL.DL.Entities;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
 
 namespace CRUDRecipeEF.BL.DL.Services
 {
     public class RecipeService : IRecipeService
     {
         private readonly RecipeContext _context;
         private readonly IMapper _mapper;
+        private readonly ILogger<RecipeService> _logger;
```

Comment:
what is the reason to use serilog in our case? i ran the app and it seems like all its doing is the same as if we had used microsofts logger instead. i believe the built in logger also allows us to write to files

Reply:
Look at the json file in the log directory to see the advantage